ts/packages/ts/src/commands/ExportCommand.ts

```diff
@@ -1,22 +1,44 @@
 import _ from 'lodash';
 import { Command } from '../Command';
 import { type EvalCommandReturn } from './EvalCommand';
+import { NAME, ConfigError } from '../utils';
 
 export type ExportCommandReturn = {
   [key: string]: string;
 };
 
 export type ExportCommandParameters = {
   pipe: EvalCommandReturn;
+  keys?: (key: string) => string;
 };
 
 export class ExportCommand extends Command<ExportCommandReturn> {
   constructor(public parameters: ExportCommandParameters) {
     super(parameters);
   }
 
+  private mutateKeys(result: EvalCommandReturn): EvalCommandReturn {
+    const { keys } = this.parameters;
+
+    if (!keys) {
+      return result;
+    }
+
+    return _.mapKeys(result, (current, key) => {
+      try {
+        const mutatedKey = keys(key);
+        if (!NAME(mutatedKey)) {
+          throw new ConfigError('invalid config key', `${mutatedKey} must be a valid name`);
+        }
+        return mutatedKey;
```

Comment:
Let's maintain the error scoping in usage of the commands. This also aligns with the same logic in `ConfigSchema` key validation:
```node.js
  const errorScope: [string, string][] = [
        ['ExportCommand', ''],
        ['ConfigKey', key],
      ];
      const mutatedKey = keys(key);
      if (!NAME(mutatedKey)) {
        throw new ConfigError('invalid config key', `${mutatedKey} must be a valid name`, errorScope);
      }
      return mutatedKey;
```

Reply:
done